packages/network/iptables/package.mk

```diff
@@ -8,13 +8,10 @@ PKG_SHA256="993a3a5490a544c2cbf2ef15cf7e7ed21af1845baf228318d5c36ef8827e157c"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.netfilter.org/"
 PKG_URL="http://www.netfilter.org/projects/iptables/files/$PKG_NAME-$PKG_VERSION.tar.bz2"
-PKG_DEPENDS_TARGET="toolchain linux libmnl libnftnl"
+PKG_DEPENDS_TARGET="toolchain linux:host libmnl libnftnl"
```

Comment:
Nice - this will hopefully allow some packages (connman, avahi, samba etc.) to build a little earlier.